keyboards/annepro2/boards/ANNEPRO2_C15/board.c

```diff
@@ -0,0 +1,119 @@
+/*
+    Copyright (C) 2020 Yaotian Feng, Codetector<codetector@codetector.cn>
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#include <string.h>
+#include "hal.h"
+#include "wait.h"
+#include "led.h"
+#include "annepro2.h"
```

Comment:
ChibiOS board definitions should not include keyboard files.
```suggestion
#include "hal.h"
```